guide/14-deep-learning/inference-only-text-models-in-arcgis-learn.ipynb

```diff
@@ -0,0 +1,987 @@
+{
```

Comment:
<p>reference the paper which provided this image</p><p> </p><blockquote>To get a more detail explanation on **different forms of attention** visit [this](https://towardsdatascience.com/attention-and-its-different-forms-7fc3674d14dc) page. Also there is a great blog post on [Visualizing attention in machine translation model](https://jalammar.github.io/visualizing-neural-machine-translation-mechanics-of-seq2seq-models-with-attention/) that can help in understanding the attention mechanism in a better way.&nbsp;An **“annotated”**[[3]](#References) version of the paper is also present in the form of a line-by-line implementation of the transformer architecture.</blockquote><p>To get a more <s>detail</s> detailed explanation on <strong>different forms of attention,</strong> visit [this](https://towardsdatascience.com/attention-and-its-different-forms-7fc3674d14dc) page. Also there is a great blog post on [Visualizing attention in machine translation model](https://jalammar.github.io/visualizing-neural-machine-translation-mechanics-of-seq2seq-models-with-attention/) that can help in understanding the attention mechanism in a better way.&nbsp;An **“annotated”**[[3]](#References) version of the paper is also present in the form of a line-by-line implementation of the transformer architecture.</p><p> </p><p> </p><p>https://jalammar.github.io/illustrated-transformer/ should this blog also be mentioned </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/851/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Made the necessary changes</p>